framework/src/executioners/FixedPointSolve.C

```diff
@@ -71,12 +73,31 @@ FixedPointSolve::validParams()
                                     "during fixed point iterations. This check is "
                                     "performed based on postprocessor defined by "
                                     "the custom_pp residual.");
-  params.addParam<bool>("direct_pp_value",
-                        false,
-                        "True to use direct postprocessor value "
-                        "(scaled by value on first iteration). "
-                        "False (default) to use difference in postprocessor "
-                        "value between fixed point iterations.");
+
+  params.addParamNamesToGroup(
+      "accept_on_max_fixed_point_iteration fixed_point_rel_tol fixed_point_abs_tol "
+      "custom_pp direct_pp_value custom_abs_tol custom_rel_tol",
+      "Fixed point iterations");
+
+  return params;
+}
+
+InputParameters
+FixedPointSolve::validParams()
+{
+  InputParameters params = emptyInputParameters();
+  params += FixedPointSolve::fixedPointDefaultConvergenceParams();
+
+  params.addParam<unsigned int>(
+      "fixed_point_min_its", 1, "Specifies the minimum number of fixed point iterations.");
+  params.addParam<unsigned int>(
+      "fixed_point_max_its", 1, "Specifies the maximum number of fixed point iterations.");
```

Comment:
I thought you wanted to move these to the convergence object too?

Reply:
`FixedPointSolve` needs at least the max one to determine if it's even going to do a loop. For instance, only if > 1, it'll print "Beginning fixed point iteration 0".